### PR TITLE
fix ban spam when someone joins and spams at 308 mph

### PR DESCRIPTION
**Pull request by Deauthorized**
there's probably a more efficient and less retarded way of doing this but its 2 am and i cannot think

also added a warning for when you're halfway to the spam threshold (4 mp/s)

### CODE REVIEW

**Comment by Deauthorized** [comment]
if you're wondering what ban spam is, pretty much if you spam too fast it will call eject multiple times leading to players becoming day banned for accidentally leaving spammer on

this is no longer apparent

**Comment by ajaxvp** [comment]
i'll review this commits later. it looks fine, but i'd like to change a few things.

File: src/main/java/me/totalfreedom/totalfreedommod/AntiSpam.java [hunk 1]
```diff
@@ -114,4 +133,22 @@ public void onPlayerCommandPreprocess(PlayerCommandPreprocessEvent event)
...
+        if (markedfordeath.contains(event.getPlayer().getName()))
+        {
+            markedfordeath.remove(event.getPlayer().getName());
+        }
+    }
+    
+    @EventHandler(priority = EventPriority.NORMAL)
+    public void onPlayerLogin(PlayerLoginEvent event)
+    {
+        if (event.getPlayer().getName().startsWith("McStorm_"))
```
Comment (ncurran02) [inline src/main/java/me/totalfreedom/totalfreedommod/AntiSpam.java]: Not ideal

Comment (ajaxvp) [reply src/main/java/me/totalfreedom/totalfreedommod/AntiSpam.java]: ye

Comment (Deauthorized) [reply src/main/java/me/totalfreedom/totalfreedommod/AntiSpam.java]: added that cuz there were bots with "mcstorm" as a prefix, you can remove it if you want idrc

**Pull request merged**